Finix/Presentation/Components/Profile/AvatarView.swift

```diff
@@ -0,0 +1,42 @@
+// Created by Valerie N. Prinz on 20/11/2024.
+
+import SwiftUI
+
+extension AvatarView {
+    struct Model {
+        enum Size {
+            case small, big
+        }
+        
+        let imageName: String
+        let size: Size
+        
+        var imageSize: CGFloat {
+            switch size {
+            case .small: 48
+            case .big: 64
+            }
+        }
+    }
+}
+
+struct AvatarView: View {
+    let model: Model
+    
+    var body: some View {
+        Image(model.imageName)
```

Comment:
There is no need in a separate component for a default image.
The logic should be the following:
- There is an AvatarView
- By default, an "avatar" is shown
- If model contains an image, it should be shown instead
- When user changes the avatar, model should be updated -> View should be redrawn

```suggestion
        Image(model.imageName ?? "avatar")
```

Reply:
Yes. If there is an asset to use, we don't need to put additional logic.